% !TEX program = xelatex
\documentclass[a4paper,10pt]{report}

\pagestyle{plain}
% \pagenumbering{gobble}
\pagenumbering{arabic}

% \setcounter{page}{215}

%A Few Useful Packages
\usepackage{marvosym}
\usepackage{fontspec} 					%for loading fonts
\usepackage{xunicode,xltxtra,url,parskip} 	%other packages for formatting
\usepackage{setspace, lipsum}	% packages for line spacing
\RequirePackage{color,graphicx}
\usepackage[usenames,dvipsnames]{xcolor}
\usepackage[big]{layaureo} 				%better formatting of the A4 page
% an alternative to Layaureo can be ** \usepackage{fullpage} **
\usepackage{supertabular} 				%for Grades
\usepackage{titlesec}					%custom \section
%Setup hyperref package, and colours for links
\usepackage{hyperref}
\usepackage{longtable}
\usepackage[none]{hyphenat}
\usepackage{ragged2e}
\definecolor{linkcolour}{rgb}{0,0.2,0.6}
\hypersetup{colorlinks,breaklinks,urlcolor=linkcolour, linkcolor=linkcolour}
%FONTS
\usepackage[sfdefault]{inter}
\defaultfontfeatures{Mapping=tex-text}
\setmainfont{Inter} %Set font to Helvetica

\usepackage[outline]{contour}
\contourlength{2pt}
\contournumber{10}

%CV Sections inspired by:
%http://stefano.italians.nl/archives/26
\titleformat{\section}{\Large\scshape\raggedright}{}{0em}{}[\titlerule]
\titlespacing{\section}{0pt}{0pt}{0pt}

%Italian hyphenation for the word: ''corporations''
\hyphenation{im-pre-se}

%-------------WATERMARK TEST [**not part of a CV**]---------------
\usepackage[absolute]{textpos}
\usepackage{longtable}
\setlength{\TPHorizModule}{30mm}
\setlength{\TPVertModule}{\TPHorizModule}
\textblockorigin{1mm}{0.65\paperheight}
\setlength{\parindent}{0pt}

\usepackage{enumitem}
\setlist[itemize]{topsep=0pt, partopsep=0pt, align=parleft, left=0pt..1em}

%-------------Document Size -----------------------
\usepackage{geometry}
\geometry{ bmargin=1in, tmargin = 1in}

%--------------------BEGIN DOCUMENT----------------------
\begin{document}

% \pagestyle{empty} % non-numbered pages

% Thesis adjustments


\font\fb=''[cmr10]'' %for use with \LaTeX command

%--------------------TITLE-------------
\par{
	\centering{
		\Huge Dakota Y. \textsc{Hawkins}
	}\bigskip\par
}

%--------------------SECTIONS-----------------------------------
%Section: Personal Data
\section{\color{linkcolour}{Contact}}

\begin{tabular}{rl}
	\textsc{e-mail:}  & \href{mailto:dakotahawkins0110@gmail.com}{dakotahawkins0110@gmail.com}      \\
	\textsc{GitHub:}  & \href{https://github.com/dakota-hawkins}{https://github.com/dakota-hawkins} \\
	\textsc{website:} & \href{https://dakota-hawkins.github.io}{https://dakota-hawkins.github.io}
\end{tabular}

%Section: Education
\section{\color{linkcolour}{Education}}
\begin{tabular}{rp{11.5cm}}
	\textsc{2016 -- 2023} & Doctor of Philosophy, \textbf{Boston University}, Boston, MA                                                                                                                                                 \\
	                      & Bioinformatics | Cynthia A. Bradham Laboratory                                                                                                                                                               \\
	                      & \RaggedRight{ \small \textbf{Thesis}: \emph{\footnotesize{Understanding Cell-Type Diversification During Developmental Pattern Formation in Sea urchin Embryos Using Single Cell and Molecular Approaches}}} \\

	\textsc{2010 -- 2015} & Bachelor of Science, \textbf{Westminster College}, Salt Lake City, UT                                                                                                                                        \\
	                      & \emph{cum laude} | Majors: Biology and Mathematics                                                                                                                                                           \\
	                      & \RaggedRight{\small \textbf{Research}: signal processing to detect anomalous singing on the nest in Mockingbirds and characterize urban vs non-urbon in Finch song dialects}
\end{tabular}

%Section: Research
\section{\color{linkcolour}{Recent Professional History}}
% \begin{tabular}{rp{10cm}}
\begin{longtable}{r|p{10.5cm}}
	\textsc{May 2017 -- Jun. 2023}  & \textbf{Boston University}, Boston, MA                                         \\
	                                & \small \emph{Doctoral Student} - Bradham Lab                                   \\
	                                & \footnotesize{
		\vspace{-3.5mm}
		% \hspace{-5mm}
		\begin{itemize}
			\setlength\itemsep{0em}
			% \setlength\leftmargini{-5mm}
			% \setlength\leftmarginii{-5mm}
			\item Developed a novel machine learning algorithm to accurately identify cell states in mixed-condition scRNA-seq datasets.
			\item Mechanistically characterized cell type diversification of skeletal lineage cells in sea urchins using multi-condition scRNA-seq data consisting of 5 time points, 6 experimental conditions, and 10s of thousands of cells.
			\item Developed novel computational and machine learning workflows to integrate 3D smFISH imaging data with scRNA-seq data to infer embryonic locations of scRNA-seq defined cell types.
		\end{itemize}
	} \vspace{-3.5mm}                                                                                                \\
	\multicolumn{2}{c}{}                                                                                             \\[-0.75em]
	\textsc{Jul. 2016 -- May 2017}  & \textbf{Boston University}, Boston, MA                                         \\
	                                & \small \emph{Doctoral Rotation Student} - Sebastiani, Monti, and Gallagan Labs \\
	                                & \footnotesize{
		\vspace{-3.5mm}
		\begin{itemize}
			\setlength\itemsep{0em}
			\item Performed eQTL analysis on whole-genome and bulk RNA-seq data to establish tissue-specific biomarkers for Alzheimer's disease.
			\item Determined cancer-specific immune response in
			      tumor cells by leveraging general linear models to identify key signatures in bulk RNA-seq.
			\item Conducted and analyzed ChIP-seq and RNA-seq experiments to help map the transcriptional
			      regulatory network of \emph{E.\ coli}.
		\end{itemize}
	} \vspace{-3.5mm}                                                                                                \\
	\multicolumn{2}{c}{}
	\\[-0.75em]
	\textsc{Jul. 2015 -- Jul. 2016} & \textbf{Pacific Northwest National Laboratory}, Richland, WA                   \\
	                                & \small \emph{Research Assistant} - Applied Stats. and Comp. Modeling Group     \\
	                                & \footnotesize{
		\vspace{-3.5mm}
		\begin{itemize}
			\setlength\itemsep{0em}
			\item Performed multi-omic analysis to identify key differences in metabolomic consumption and microbiome composition between gastric bypass patients.
			\item Created software workflows to visualize and quantify spliceforms in high-throughput proteomic data.
		\end{itemize}
	} \vspace{-3.5mm}                                                                                                \\
	\multicolumn{2}{c}{}                                                                                             \\[-0.75em]
\end{longtable}


%Section: Languages
\section{\color{linkcolour}{Programming Languages and Tooling}}
\begin{longtable}{rp{12cm}}
	\textsc{Python:}    & Used for data analysis, machine learning, and package development.                                    \\
	                    & \small{\href{https://github.com/BradhamLab/icat}{https://github.com/BradhamLab/icat}}                 \\
	\textsc{R:}         & Used for -omics data analysis and visualization.                                                      \\
	                    & \small{\href{https://github.com/BradhamLab/scPipe}{https://github.com/BradhamLab/scPipe}}             \\
	\textsc{Snakemake:} & Used to generate stable and modular pipeline workflows.                                               \\
	                    & \small{\href{https://github.com/BradhamLab/indrops-star}{https://github.com/BradhamLab/indrops-star}} \\
	\textsc{C++:}       & Extended existing Louvain library for semi-supervised clustering.                                     \\
	                    & \small{\href{https://github.com/BradhamLab/sslouvain}{https://github.com/BradhamLab/sslouvain}}       \\
	\textsc{git}:       & Used for version control and collaboration.                                                           \\
	                    & \small{\href{https://github.com/dakota-hawkins}{https://github.com/dakota-hawkins}}                   \\
	\textsc{conda}:     & Environment handling and package installation for reproducible analysis.                              \\
	\textsc{linux}:     & Used for analysis in a high-performance cluster as well as daily use.                                 \\
	\textsc{SQL}:       & Used to create lab databases for dataset annotation.                                                  \\
\end{longtable}

%Section: Publications
\section{\color{linkcolour}{Recent Publications}}
\begin{tabular}{rp{11cm}}
	\textsc{2023} & \emph{ICAT: A Novel Algorithm to Identify Cell-types in scRNA-seq Perturbation Experiments}                                                                   \\
	              & \small \textbf{Bioinformatics} \href{https://doi.org/10.1093/bioinformatics/btad278}{https://doi.org/10.1093/bioinformatics/btad278}                          \\
	              & \footnotesize \textbf{Hawkins DY}, Zuch DT, Huth J, Rodríguez-Sastre N, McCutcheon KR, Glick A, Lion AT, Thomas CF, Descoteaux AE, Johnson WE, and Bradham CA \\
	\textsc{2023} & \emph{Voltage-gated sodium channel activity mediates sea urchin larval skeletal patterning through spatial regulation of Wnt5 expression}                     \\
	              & \small \textbf{Development} \href{https://doi.org/10.1242/dev.201460}{https://doi.org/10.1242/dev.201460}                                                     \\
	              & \footnotesize Thomas CF, \textbf{Hawkins DY}, Skidanova V, Marrujo SR, Gibson J, Ye Z, and Bradham CA                                                         \\
	\textsc{2023} & \emph{Ethanol exposure perturbs sea urchin development and disrupts developmental timing}                                                                     \\
	              & \small \textbf{Developmental Biology} \href{https://doi.org/10.1016/j.ydbio.2022.11.001}{https://doi.org/10.1016/j.ydbio.2022.11.001}                         \\
	              & \footnotesize Rodríguez-Sastre N, Shapiro N, \textbf{Hawkins DY}, Lion AT, Peyreau M, Correa AE, Dionne K, and Bradham CA                                     \\
\end{tabular}

% Mentorship + Service
\section{\color{linkcolour} Mentorship and Management}
\begin{longtable}{rp{11.5cm}}
	2017 -- 2023
	 & Bradham Lab                                                                                                                                                                 \\[0.225em]
	 & \small Mentored undergraduate researchers from  project inception to presentation at national conferences.                                                                  \\

	 & \begin{itemize}
		   \vspace{-3.5mm}
		   \footnotesize{\item \emph{Optimizing Feature Selection in High-Dimensional RNA-seq Data}\newline
		         \textbf{Annual Biomedical Research Conference for Minority Students} (2021)\newline
		         Baringa ZI, \textbf{Hawkins DY}, and Bradham CA}
		   \item \emph{A Pipeline for Constructing a 3D Coordinate Map of PMCs in Developing Embryos} \newline
		         \textbf{Annual Biomedical Research Conference for Minority Students} (2021)\newline
		         \footnotesize Hughes MM, \textbf{Hawkins DY}, ..., Bradham CA
		   \item \emph{The Application of V-Net in Identifying Primary Mesenchyme Cells in Sea Urchin Embryo Images} \newline
		         \textbf{Annual Biomedical Research Conference for Minority Students} (2018)\newline
		         Price M, \textbf{Hawkins DY}, Shapiro N, Bradham CA
		         \vspace{-3.5mm}
	   \end{itemize}                                                           \\
	2017 -- 2022
	 & \textbf{\underline{B}}ioinformatics \textbf{\underline{R}}esearach and \textbf{\underline{I}}nterdisciplinary \textbf{\underline{T}}raining \textbf{\underline{E}}xperience \\[0.225em]
	 & \begin{itemize}
		   \vspace{-3.5mm}
		   \small{
		   \item Mentored Summer undergraduate researchers for the
		         BU BRITE REU.
		   \item Created, led, and scheduled summer-long workshop series on bioinformatic fundamentals including Python, R, machine learning, and workflow languages.
		   \item  Coordinated social events and extracurricular activities for visiting students.}
		         \vspace{-3.5mm}
	   \end{itemize}
	\\[-1em]
	2017 -- 2021
	 & BU Bioinformatics Student Association                                                                                                                                       \\[0.225em]
	 & \begin{itemize}
		   \vspace{-3.5mm}
		   \small{
		   \item Organized social and recruiting events for the BU
		         Bioinformatics program.
		   \item Established
		         support networks and resources for 50+ PhD students.
		   \item Established and organized quarterly meetings with program administration.
		         }
		         \vspace{-3.5mm}
	   \end{itemize}                                                                                              \\[-1em]
	2017 -- 2021
	 & First-year PhD Workshops                                                                                                                                                    \\[0.2em]                                                                                                                                                                            & \begin{itemize}
		\vspace{-3.5mm}
		\small{
		\item Organized computational workshops to quickly
		      introduce first-year PhD students to common computational tools for bioinformatic research.
		      }
		      \vspace{-3.5mm}
	\end{itemize}
	\\
	2018, 2019
	 & BU Student Organized Symposium                                                                                                                                              \\[0.2em]
	 & \begin{itemize}
		   \vspace{-3.5mm}
		   \small{
		   \item Contacted and coordinated with invited researchers.
		   \item Led day-of logistics and helped advertise the event to the broader Boston scientific community.
		         }
		         \vspace{-3.5mm}
	   \end{itemize}                                                                        \\
\end{longtable}


%Section: Awards and Accolades
\section{\color{linkcolour}{Honors and Awards}}
\begin{tabular}{rl}
	2022         & Bioinformatics Service Award                                           \\
	2020         & 1st Place Poster -- Bioinformatics Open House, Boston University       \\
	2017         & 2nd Place Poster -- IBSB Conference, Berlin Germany                    \\
	2016         & NIH Trainee Fellowship -- Boston University                            \\
	2016         & Outstanding Performance Award -- Pacific Northwest National Laboratory \\
	2014, 2015   & Honorable Mention -- Mathematical Competition in Modeling              \\
	2013 -- 2015 & Gore Math/Science Scholarship -- Wesminster College                    \\
	2013, 2014   & Gore Math/Science Summer Research Grant -- Westminster College         \\
	2012         & Scholars Summer Research Grant -- Westminster College
\end{tabular}

\end{document}

%Section: Professional Affiliations
% \section{\color{linkcolour}{Professional Affiliations}}
% \begin{tabular}{rl}
% 2014 -- Present & Beta Beta Beta (Biology Honor Society) \\
% \end{tabular}
% \end{document}
